modules/check_in/spec/request/v2/pre_check_ins_request_spec.rb

```diff
@@ -161,4 +153,48 @@
       end
     end
   end
+
+  describe 'POST `create`' do
+    let(:post_params) do
+      {
+        pre_check_in: {
+          uuid: id,
+          demographics_up_to_date: true,
+          next_of_kin_up_to_date: true,
+          check_in_type: :pre_check_in
```

Comment:
nit: "preCheckIn" instead of : pre_check_in

Reply:
updated in 2f84e5f50a810f003c289662a79e79a1d922b0ed